### ISSUE TITLE
Date Input A11y Issues

### ISSUE THREAD

**Issue by CillianHPE** [grommet/hpe-design-system#4878]
### Link to problem

https://hpe.sharepoint.com/:x:/r/teams/hpe-design/Shared%20Documents/HPE%20Design%20System/Accessibility/WCAG%20Rule%20Evaluation%20Template.xlsx?d=we975e19f32644f4cbd0a446f23f89277&csf=1&web=1&e=for2fd&nav=MTVfezAzRTJCOTg2LTUzNzAtNEZCMy1CRkZGLTM5OUNCMjVBMTMwNX0

### Current problem

Date Input fails the following WCAG Rules

- 1.1.1 (No ARIA labelling for calendar)
- 1.4.6 (Color contrast too low)
- 1.4.10 (Unable to access calendar when scrolling on small viewport)
- 1.4.12 Text spacing
-  2.1.3 Keyboard No exception (When calendar closes, the focus doesn't return to the input)
- 2.4.3 Focus Order (Failed because the calendar dates do not receive focus or get read out by the screen reader. )
- 2.5.5 Target Size enhanced

**Comment by jcfilben** [comment]
@CillianHPE 
- 1.1.1 is marked as conditional so I don't think we need to include it in this list. 
- 1.4.6 is a AAA rule so for now let's not worry about meeting it. 
- 1.4.12 is passing. 
- When calendar closes, the focus doesn't return to the input. This is something we should fix but it isn't tied to issue 2.1.3. I think it is related to 2.4.3
- 2.5.5 let's not worry about this for now since it is a AAA rule

**Comment by jcfilben** [comment]
Could we create one issue per wcag rule, this makes it easier to break up the work, priotitize it and assign it to someone. Also I think these issues should go on the [grommet repo](https://github.com/grommet/grommet) rather than the design system repo since they are related to grommet components.

**Comment by CillianHPE** [comment]
Closed as these issues were accidently created in the wrong repo.
